client_test.go

```diff
@@ -703,6 +707,31 @@ func TestClient_DialWithContextOptions(t *testing.T) {
 	}
 }
 
+// TestClient_DialWithContextOptionDialContextFunc tests the DialWithContext method plus
+// use dialContextFunc option for the Client object
+func TestClient_DialWithContextOptionDialContextFunc(t *testing.T) {
```

Comment:
I'm not sure this test works enough or not because I cannot run in my local. :bow: It needed to set `TEST_HOST`.

Reply:
Local tests succeeded.